enhancements/installer/cluster-api-based-installations.md

```diff
@@ -0,0 +1,347 @@
+`cluster-api` directory alongside the existing `manifests` and `openshift` directories:
+
+```shell=
+$ ./openshift-install create manifests --dir install-dir
+INFO Credentials loaded from the "default" profile in file "~/.aws/credentials" 
+INFO Consuming Install Config from target directory 
+INFO Manifests created in: install-dir/cluster-api, install-dir/manifests and install-dir/openshift 
+$ tree install-dir/cluster-api/
+install-dir/cluster-api/
+├── 00_capi-namespace.yaml
+├── 01_aws-cluster-controller-identity-default.yaml
+├── 01_capi-cluster.yaml
+├── 02_infra-cluster.yaml
+├── 10_inframachine_mycluster-6lxqp-master-0.yaml
+├── 10_inframachine_mycluster-6lxqp-master-1.yaml
+├── 10_inframachine_mycluster-6lxqp-master-2.yaml
+├── 10_inframachine_mycluster-6lxqp-master-bootstrap.yaml
+├── 10_machine_mycluster-6lxqp-master-0.yaml
+├── 10_machine_mycluster-6lxqp-master-1.yaml
+├── 10_machine_mycluster-6lxqp-master-2.yaml
+└── 10_machine_mycluster-6lxqp-master-bootstrap.yaml
+
+1 directory, 12 files
+```
+
+The manifests within this `cluster-api` directory will not be written to the cluster or included in bootstrap ignition.
+In future work, we expect these manifests to be pivoted to the cluster to enable the target cluster to take over managing
+its own infrastructure.
+
+The Cluster API manifests will be generated with `.spec` fields in the [manifest asset target][asset] with `.status`
+updated after cluster creation.
+
+### Risks and Mitigations
+
+While we do not expect these changes to introduce a significant security risk, we are working with product security teams
+to ensure they are aware of the changes and are able to review.
+
+
+### Drawbacks
+
+By depending on CAPI providers whose codebases live in a repository external to the Installer,
+the process for developing features and delivering fixes is more complex than in a monolothic repo. 
+While the same could be true for the Installer Terraform dependency; the CAPI providers will 
+be more actively developed than their Terraform counterparts. Furthermore, it will be necessary
+to ensure that the CAPI providers used by the Installer match the version of those in the payload.
+
+While this external dependency is a significant drawback, it is not unique to this design
+and is common throughout OpenShift (e.g. any time the API or library-go must be updated
+before being vendored into a component). To minimize the devex friction, we will focus
+on documenting a workflow for developing providers while working with the Installer.
+
+Additionally, we will explore designs to solve or mitigate these issues.
+
+## Design Details
+
+### Open Questions [optional]
+
+1. UX design during install process as well as during failure (log collection). The Installer will dump
+(potentially prettified) controller logs. Once we reach a certain level of stability it may be worthwhile
+to implement a UI.
```

Comment:
I intended TUI rather than GUI. We were dumping Terraform output straight to stdout, now we're dumping controller logs, which are less pretty and perhaps somewhat overwhelming for users. We will want to figure out how to make that experience better for users.